packaging/osx/buildpackage.sh

```diff
@@ -81,8 +81,8 @@ modify_plist "{JOIN_SERVER_URL_SCHEME}" "openra-${MOD_ID}-${TAG}" "${PACKAGING_O
 
 echo "Packaging zip archive"
 
-zip "${PACKAGING_INSTALLER_NAME}-${TAG}" -r -9 "${PACKAGING_OSX_APP_NAME}" --quiet --symlinks
-mv "${PACKAGING_INSTALLER_NAME}-${TAG}.zip" ${OUTPUTDIR}
+zip "${PACKAGING_INSTALLER_NAME}-${TAG}-osx" -r -9 "${PACKAGING_OSX_APP_NAME}" --quiet --symlinks
+mv "${PACKAGING_INSTALLER_NAME}-${TAG}-osx.zip" ${OUTPUTDIR}
```

Comment:
Why should the portable package which works on all platforms require a suffix if the osx one doesn't?!? It should be the other way around...